psalm.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<psalm
+        totallyTyped="true"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="https://getpsalm.org/schema/config"
+        xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+>
+    <projectFiles>
+        <directory name="src"/>
+        <directory name="test"/>
+        <ignoreFiles>
+            <directory name="vendor"/>
+        </ignoreFiles>
+    </projectFiles>
+
+    <issueHandlers>
```

Comment:
We should generate a baseline instead of suppressing issues by default

Reply:
@boesing This is from the suggested template: https://github.com/mezzio/repo-template/blob/6124090610ef1c6d6b0c80a063cafacc3ec68f38/psalm.xml.dist

Not sure if needed when the phpunit plugin for psalm is used. I guess we can drop them.

I'm not done yet, my goal is to not need a baseline.

Reply:
Sounds good. I've created that issue template before I've implemented psalm to any package, so yes, sadly its quite outdated. However, thanks for being so passionate :-)

Reply:
While working on this, it would require some major rewrites, especially the config handling.
So I'll fix as much as possible but try to keep the changeset low and target the rest with separate PR's

Refactoring the config handling from arrays to types is on my tasklist anyways but out of scope for this PR.

Reply:
Oh and getting rid of the baseline in the end is still the final goal ;-)